stores/iscn.ts

```diff
@@ -0,0 +1,98 @@
+import { defineStore } from 'pinia'
+
+import { getSigningClient } from '~/utils/cosmos'
+
+function addIDToRecords (records: any[]) {
+  return records.map(r => ({ id: r.data['@id'] as string, ...r }))
+}
+
+export const useIscnStore = defineStore('iscn', {
+  state: () => ({
+    recordsById: {} as any,
+    records: [],
```

Comment:
Would this be a computed property of `Object.values(recordsById)`?

Reply:
[✨ Add getter for all records and remove unused function](https://github.com/likecoin/nft-book-press/pull/317/commits/db69b8620e35b4e2b8148d81c251ede51413a290)